models/meshmodel/registry/registry.go

```diff
@@ -116,23 +119,26 @@ func (rm *RegistryManager) Cleanup() {
 func (rm *RegistryManager) RegisterEntity(h Host, en Entity) error {
 	switch entity := en.(type) {
 	case v1alpha1.ComponentDefinition:
-		isAnnotation, _ := entity.Metadata["isAnnotation"].(bool)
-		if entity.Schema == "" && !isAnnotation { //For components which an empty schema and is not an annotation, exit quietly
+		if entity.Schema == "" { //For components with an empty schema, exit quietly
```

Comment:
Because I asked a few times whether we should count those entities which are with empty schema lee replied me with [this](https://layer5io.slack.com/archives/CFGG6U10E/p1704404768819809?thread_ts=1704404768.819809&cid=CFGG6U10E) 

I might have misunderstood it.

Reply:
The actor registering these should be responsible and should maintain how many of the models comps (schema or schema-less) were registered and how many/ which of them failed.